targets/xiao.json

```diff
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "xiao"],
+    "flash-1200-bps-reset": "true",
+    "flash-method": "msd",
+    "msd-volume-name": "ITSYBOOT",
```

Comment:
It looks like they just copied the ItsyBitsy-M0 bootloader, which is unfortunate. But we'll have to match that.

Reply:
Fixed at 201dfc2aa883091b6bcacd9c31ff6d69e7e9ec74
The volume name was `Arduino` as well as Wio Terminal (#1124).